test/run_tests.py

```diff
@@ -36,9 +36,9 @@
 
 Examples:
 
-  run_tests.py python chrome
-  run_tests.py jython firefox --suite list
-  run_tests.py python chrome --scusername your_username --sckey account_key
+  run_tests.py chrome
+  run_tests.py --suite list --interpreter jython firefox
```

Comment:
The `list` was a bad example, because it refers to `list.robot` file and not to Python list. Changed to example to `javascript`